packages/expect/src/__tests__/matchers.property.test.ts

```diff
@@ -0,0 +1,200 @@
+    fc.assert(
+      fc.property(fc.dedup(fc.anything(anythingSettings), 2), ([a, b]) => {
+        // Given:  a and b identical values
+        // Assert: We expect `expect(a).toStrictEqual(b)`
+        expect(a).toStrictEqual(b);
+      }),
+      assertSettings,
+    );
+  });
+
+  it('should be symmetric', () => {
+    const safeExpectStrictEqual = (a, b) => {
+      try {
+        expect(a).toEqual(b);
+        return true;
+      } catch (err) {
+        return false;
+      }
+    };
+    fc.assert(
+      fc.property(
+        fc.anything(anythingSettings),
+        fc.anything(anythingSettings),
+        (a, b) => {
+          // Given:  a and b values
+          // Assert: We expect `expect(a).toStrictEqual(b)`
+          //         to be equivalent to `expect(b).toStrictEqual(a)`
+          expect(safeExpectStrictEqual(a, b)).toBe(safeExpectStrictEqual(b, a));
+        },
+      ),
+      assertSettings,
+    );
+  });
+});
+
+describe('.toEqual()', () => {
+  it('should be reflexive', () => {
+    fc.assert(
+      fc.property(fc.dedup(fc.anything(anythingSettings), 2), ([a, b]) => {
+        // Given:  a and b identical values
+        // Assert: We expect `expect(a).toEqual(b)`
+        expect(a).toEqual(b);
+      }),
+      assertSettings,
+    );
+  });
+
+  it('should be symmetric', () => {
+    const safeExpectEqual = (a, b) => {
+      try {
+        expect(a).toEqual(b);
+        return true;
+      } catch (err) {
+        return false;
+      }
+    };
+    fc.assert(
+      fc.property(
+        fc.anything(anythingSettings),
+        fc.anything(anythingSettings),
+        (a, b) => {
+          // Given: a and b values
+          // Assert: We expect `expect(a).toEqual(b)`
+          //         to be equivalent to `expect(b).toEqual(a)`
+          expect(safeExpectEqual(a, b)).toBe(safeExpectEqual(b, a));
+        },
+      ),
+      assertSettings,
+    );
+  });
+});
+
+describe('.toContain()', () => {
+  it('should always find the value when inside the array', () => {
+    fc.assert(
+      fc.property(
+        fc.array(fc.anything(anythingSettings)),
+        fc.array(fc.anything(anythingSettings)),
+        fc.anything(anythingSettings).filter(v => !Number.isNaN(v)),
+        (startValues, endValues, v) => {
+          // Given:  startValues, endValues arrays and v value (not NaN)
+          // Assert: We expect `expect([...startValues, v, ...endValues]).toContainEqual(v)`
+          expect([...startValues, v, ...endValues]).toContain(v);
+        },
+      ),
+      assertSettings,
+    );
+  });
+
+  it('should not find the value if it has been cloned into the array', () => {
+    fc.assert(
+      fc.property(
+        fc.array(fc.anything(anythingSettings)),
+        fc.array(fc.anything(anythingSettings)),
+        fc.dedup(fc.anything(anythingSettings), 2),
+        (startValues, endValues, [a, b]) => {
+          // Given:  startValues, endValues arrays
+          //         and [a, b] identical values

```

Comment:
`identical` sounds like object identity to me, but they're just structurally equal. Maybe
`[a, b] equal, but not the same values`